examples/2d/text2d.rs

```diff
@@ -47,10 +47,29 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     // Demonstrate changing scale
     commands
         .spawn_bundle(Text2dBundle {
-            text: Text::with_section("scale", text_style, text_alignment),
+            text: Text::with_section("scale", text_style.clone(), text_alignment),
             ..default()
         })
         .insert(AnimateScale);
+    // Demonstrate text wrapping
+    commands.spawn_bundle(SpriteBundle {
+        sprite: Sprite {
+            color: Color::rgb(0.25, 0.25, 0.75),
+            custom_size: Some(Vec2::new(300.0, 200.0)),
+            ..default()
+        },
+        transform: Transform::from_xyz(0.0, -250.0, 0.0),
+        ..default()
+    });
+    commands.spawn_bundle(Text2dBundle {
+        text: Text::with_section("this text wraps in the box", text_style, text_alignment),
+        text_2d_bounds: Text2dBounds {
+            // Wrap text in the rectangle
+            size: Size::new(300.0, 200.0),
+        },
+        transform: Transform::from_xyz(0.0, -250.0, 1.0),
+        ..default()
+    });
```

Comment:
Could you use variables for the bounds so that they are reused for the sprite bundle and the text bounds? That would make it easier for someone who wants to play with the example

Reply:
OK. I also changed the text alignment to top-left here, as this is probably a more useful case.